graphql-spqr-spring-boot-autoconfigure/src/main/java/io/leangen/graphql/spqr/spring/web/apollo/ApolloMessage.java

```diff
@@ -81,8 +83,12 @@ public static TextMessage keepAlive() throws JsonProcessingException {
         return jsonMessage(KEEP_ALIVE);
     }
 
+    public static TextMessage connectionError(final String message) throws JsonProcessingException {
```

Comment:
These are all fair. I'm just curious how are using these? They're only ever called from `ApolloProtocolHandler` internally. Are you overriding that or using these outside of Apollo protocol?

Reply:
Yeah I actually wrote a web socket handler that does JWT based authentication, it gets the Authorization through the Apollo message payload. My usecase is so that I can easily send Apollo-compliant messages to the client in case of failure in authentication or missing the Authorization entry in the payload